packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -300,6 +300,22 @@ export class Account {
     return await this.client.call('listaccounthistory', [owner, options], 'number')
   }
 
+  /**
+   * Returns information about single account history
+   *
+   * @param {string} [owner] Single account ID (CScript or address)
+   * @param {number} [blockHeight] Height to iterate from genesis block
+   * @param {number} [txn] Order in block
```

Comment:
the bracket is actually jsdoc optional syntax

```suggestion
   * @param {string} owner Single account ID (CScript or address)
   * @param {number} blockHeight Height to iterate from genesis block
   * @param {number} txn Order in block
```